Roommates.cs

```diff
@@ -48,6 +48,16 @@ private static int CountChairs()
 			for (int j = WorldGen.roomY1; j < WorldGen.roomY2; j++) {
 				if (Main.tile[i, j].TileType == TileID.Chairs) {
 					numChairs++;
+					continue;
+				}
+
```

Comment:
I think this part can be removed, because `CountsAsChair` already has TileID 15 (which is `Chairs`) in the array.
```suggestion
```

Reply:
Good Catch!